### PR TITLE
[Review] Refactor event listeners

### LINKED ISSUE #144: support other handlers
Looks like the unsupported ones are:

- [ ] [`handleBeforeInput`](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handlebeforeinput)
- [ ] [`handlePastedText`](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handlepastedtext)
- [ ] [`handlePastedFiles`](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handlepastedfiles)
- [ ] [`handleDroppedFiles`](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handledroppedfiles)
- [ ] [`handleDrop`](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handledrop)

### PR DESCRIPTION
**Pull request by juliankrispel**
A first pr to change how event listeners are attached to Editor

fixes #144 

cc @nikgraf @ianstormtaylor

### CODE REVIEW

**Comment by nikgraf** [comment]
@juliankrispel can you check why travis fails?

**Comment by juliankrispel** [comment]
bloody style :)

I squashed the commits into one...

File: draft-js-plugins-editor/src/Editor/index.js [hunk 1]
```diff
@@ -219,8 +153,53 @@ export default class PluginEditor extends Component {
...
+  createOnListener = (name) => (event) => (
+    this.plugins
+      .filter(plug => typeof plug[name] === 'function')
+      .forEach(plug => plug[name](event))
+  );
+
+  createEventListeners = () => {
+    const listeners = {
+      onChange: this.onChange,
+      editorState: this.editorState,
```
Comment (ianstormtaylor) [inline draft-js-plugins-editor/src/Editor/index.js]: Doesn't make sense for `editorState` to be in the listeners section since it's just data.

Comment (juliankrispel) [inline draft-js-plugins-editor/src/Editor/index.js]: fair point, the variable name should be different

**Pull request merged**